Automattic/wp-calypso
# G Suite: Remove G Suite Stats upsell banner code

#### Changes proposed in this Pull Request

This is follow-up to #40338. This PR removes code references to `GSuiteStatNudge` component and selectors etc

#### Testing instructions



* Checkout this branch
* Start your development server i.e. http://calypso.localhost:3000
* Add G Suite to your cart
* Checkout/purchase
* Confirm that the there are no errors in the browser console, or server console
